composer.json

```diff
@@ -26,8 +26,8 @@
         "monolog/monolog": "^2.3",
         "ondrejmirtes/better-reflection": "^4.3.81",
         "phpstan/phpstan": "^1.2",
-        "spryker-sdk/evaluator": "dev-bugfix/sdk-302-fixed-error-return as 0.1.0",
-        "spryker-sdk/sdk-contracts": "dev-master",
+        "spryker-sdk/evaluator": "dev-feature/sdk-321-reports-impovments as 0.1.0",
```

Comment:
Please check this PR
https://github.com/spryker-sdk/sdk/pull/64/files
This lib will be renamed shortly